sycl/plugins/level_zero/tracing.cpp

```diff
@@ -16,7 +16,7 @@
 #include <layers/zel_tracing_api.h>
 #include <ze_api.h>
 
-#include <sycl/detail/iostream_proxy.hpp>
+#include <../../source/detail/iostream_proxy.hpp>
```

Comment:
I'm pretty sure this is wrong. What's the correct way to share an include between source and plugin?